delete-netlify/index.js

```diff
@@ -0,0 +1,74 @@
+const axios = require("axios");
+const fs = require("node:fs");
+const Bluebird = require("bluebird");
+
+const NETLIFY_ACCESS_TOKEN = "YOUR TOKEN HERE";
```

Comment:
can put in env?

Reply:
tried, doesn't work - not entirely sure why too? i ran `source .env` then `node <script>` but the token was undefined

Reply:
Oh that's really weird, can use dotenv? should auto pick-up from .env without source

Reply:
can i just leave as-is? doesn't impact usability and not much value add from fixing